sdk/resourcemanager/azure-resourcemanager/CHANGELOG.md

```diff
@@ -2,13 +2,9 @@
 
 ## 2.15.0-beta.1 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
```

Comment:
Should this be `Bug fixed`?

Reply:
fixed, thanks.

maybe seen too many breaking changes recently on appplatform and netowrk...